classes-metodos-atributos/exercicio-01/src/main/java/MyProgram.java

```diff
@@ -1,7 +1,33 @@
-public class MyProgram {
+import java.util.Scanner;
 
+/**
+  * MyProgram
+  */
+ public class MyProgram {
+ 
+ 
     public static void main(String[] args) {
-        System.out.println("My First Program!");   
+        
+        Scanner in = new Scanner (System.in);
```

Comment:
Verificar se é preciso fechar o in. Aparece um warning no repositório.

Reply:
Não precisa fechar o "in", está correto a chamada da função Scanner in = new Scanner (System.in). O java executa este programa perfeitamente.

Reply:
Essa é a mensagem que está na aba PROBLEMAS/PROBLEMS do VSCode: Resource leak: 'in' is never closed.
Então, sim, precisa ser fechada e o código precisa vir sem nenhum PROBLEMA relatado nessa aba. 
Faça as verificações antes de enviar.

Reply:
Para solucionar o erro do in, na função Scanner, inserir in.close();